docs/source/en/api/pipelines/flux.md

```diff
@@ -355,8 +355,74 @@ image.save('flux_ip_adapter_output.jpg')
     <figcaption class="mt-2 text-sm text-center text-gray-500">IP-Adapter examples with prompt "wearing sunglasses"</figcaption>
 </div>
 
+## Optimize
 
-## Running FP16 inference
+Flux is a very large model and requires ~50GB of RAM/VRAM to load all the modeling components. Enable some of the optimizations below to lower the memory requirements.
+
+### Group offloading
```

Comment:
@stevhliu it might be a good idea to consider making it more generally available to all major pipelines with high usage.